src/scrot.c

```diff
@@ -400,38 +400,54 @@ Window scrotGetWindow(Display *display, Window window, int x, int y)
 void scrotGrabMousePointer(const Imlib_Image image, const int xOffset,
     const int yOffset)
 {
-    XFixesCursorImage *xcim = XFixesGetCursorImage(disp);
-
-    if (!xcim) {
-        warnx("Failed to get mouse cursor image.");
-        return;
+    XFixesCursorImage *xcim = NULL;
+    char *errmsg = NULL;
+    unsigned short width;
+    unsigned short height;
+    int x;
+    int y;
+    size_t pixcnt;
+    uint32_t *pixels = NULL;
+    Imlib_Image imcursor;
```

Comment:
Aside from the pointers which gets freed at the cleanup block, why not declare rest of the variables where they are actually initialized.

Reply:
No reason, just a style choice.

Reply:
Hi again, is keeping the original location of the declarations important or is it okay if this PR changes them?

Reply:
No clue, I'm not the maintainer here :shrug: